Build.md

```diff
@@ -2,100 +2,198 @@
 
 Follow these instructions if you want to get started developing on TrenchBroom.
 
-To see how releases of TB are packaged, consult our CI scripts instead.
+To see how releases of TrenchBroom are packaged, consult our CI scripts instead.
 
 ## All Platforms
 
 First, clone the TrenchBroom repository. If you are using the official repository and not a fork, you can clone the
 repository by running
 
-```
+```bash
 git clone --recursive https://github.com/kduske/TrenchBroom.git
 ```
 
+---
+
 ## Windows
 
-- Visual Studio 2017 or 2019 can be used for development. Code needs to compile with VS 2017 (that's what our releases are built with). 
-- For Visual Studio 2017:
-    - In the Visual Studio Installer, you'll need to install:
-      - Workloads 
-        - **Desktop development with C++**
-      - Individual components
-        - **VC++ 2017 version 15.9 v14.16 latest v141 tools**
-        - **Windows Universal CRT SDK**
-        - **Windows XP support for C++**
-  - For Visual Studio 2019:
-    - In the Visual Studio Installer, you'll need to install:
-        - Workloads
-          - **Desktop development with C++**
-  - Download and install [Qt](https://www.qt.io/download) for MSVC 2017 32-bit
-  - Download and install [CMake](http://www.cmake.org) for Windows
-  - Download and install [pandoc](http://www.pandoc.org)
-  - Create a subdirectory of the TrenchBroom directory, e.g. "build", and open a command prompt there
-  - For Visual Studio 2017, run:
-
-    ```
-    cmake .. -T v141 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=C:\Qt\5.13.0\msvc2017
-    ```
-
-  - For Visual Studio 2019, run:
-
-    ```
-    cmake .. -G "Visual Studio 16 2019" -A Win32 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=C:\Qt\5.13.0\msvc2017
-    ```
-
-    Finally, open `build\TrenchBroom.sln`.
-
-  - **Note:** specifying CMAKE_BUILD_TYPE is required when running cmake; you can't change between Release and Debug from Visual Studio
+Visual Studio 2017 or 2019 can be used for development. TrenchBroom needs to compile with VS 2017 (that's what our releases are built with).
+
+### Prerequisites
+
+In order to develop, debug and compile TrenchBroom, you need to install tools listed below.
+
+#### Visual Studio 2017
+
+Download [VS 2017](https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2017-family) and install it.
+
+You'll also need to install these dependencies using Visual Studio Installer (included with main installer):
+
+- Workloads 
+  - **Desktop development with C++**
+- Individual components
+  - **C++ Windows XP Support for VS 2017 (v141) tools**
+  - **MSVC v141 - VS 2017 C++ x64/x86 build tools (v14.16)**
+  - **Windows Universal CRT SDK**
```

Comment:
Are all of these actually required? I'm pretty sure XP Support is not required, and I can build TB just fine without Universal CRT in VS2019.

Reply:
@ericwa Can you confirm?

Reply:
Checking now. I'm doing a new install of VS2017 Community with just "Desktop development with C++"

Reply:
OK, none of those "Individual components" are required on VS2017. so it should just say:

- Workloads 
  - **Desktop development with C++**